src/db/tables/subscriber_churns.ts

```diff
@@ -32,4 +32,18 @@ async function upsertSubscriberChurns(
   }
 }
 
-export { upsertSubscriberChurns };
+async function getAllSubscriberChurns(): Promise<SubscriberChurnRow[]> {
+  try {
+    const res = await knex("subscriber_churns").select().returning("*");
+
+    logger.info("get_all_subscriber_churns_success", { count: res.length });
+    return res as SubscriberChurnRow[];
```

Comment:
Also, this shouldn't need a type assertion, which can hide issues.

```suggestion
    return res;
```

(I think the `returning` is breaking type inference.)